### DISCUSSION TITLE
[nestjs-supabase-auth]Passport Strategy for Nest.js

### DISCUSSION THREAD

**Discussion by hiro1107** [supabase/supabase#502] [Feature Requests]
Hi, I just released an NPM package called **nestjs-supabase-auth**. It is a Passport Strategy for Nest.js. 

https://github.com/hiro1107/nestjs-supabase-auth

To be honest, this is the first time for me to publish an NPM package, so there should be many things that can be improved in the code and documentations. 

If you are building Nest.js API server and Supabase, try out the package and give some feedback! Any feedback is welcome!

**Comment by kiwicopple** [comment]
Wow, this is awesome @hiro1107 ! 

TBH I've never tried NestJS, although it looks great. If you want, you can create a `nestjs-todo-list` example (https://github.com/supabase/supabase/tree/master/examples) using your library. It would be a good way to showcase it (we plan to add Walkthroughs for all of the "todo" examples)

**Reply by johannesschobel** [reply]
Dear @hiro1107 ,

thank you very much. i would highly appreciate a demo application with nestjs and supabase.
Would you create and share one? That would be awesome.

All the best
